swagger.yaml

```diff
@@ -246,6 +246,55 @@ paths:
           description: Wrong content type (should be plain text)
         422:
           description: Missing fields
+  /projects/{projectId}/collaboration:
+    put:
```

Comment:
I think POST `/projects/{projectId}/share` would be more adequate

Reply:
Sounds good, will change